operator/src/main/java/oracle/kubernetes/operator/rest/model/AdmissionRequest.java

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2022, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator.rest.model;
+
+import java.util.Map;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * AdmissionRequest represents a Kubernetes admission request inside an AdmissionReview object sent by the
+ * Kubernetes ApiServer upon invoking an admission webhook. It describes the details of request, including
+ * the information avout the resource that the required admission call is targeted to, as well as the existing
+ * version and the proposed version of the resource.
+ *
+ * <p>More info:
+ * <a href="https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/
+ #webhook-request-and-response">Admission webhook request and response</a>.
+ * </p>
+ */
+public class AdmissionRequest {
+  /**
+   * An uid uniquely identifying this admission call.
+   */
+  @SerializedName("uid")
+  @Expose
+  protected String uid;
+
+  /**
+   * Fully-qualified group/version/kind of the incoming object.
+   */
+  @SerializedName("kind")
```

Comment:
I'm not following this. How is a map a "fully-qualified group/version/kind" of an object?

Reply:
The map contains the <key, value> pairs for "group", "version" and "kind" of the object.